### PR TITLE
Fix 1644

### LINKED ISSUE #1644: weighing-machine: add task to verify Units property is initialized to Units.Kilograms
The `weighing-machine` tests depend on the `Units` property being initialized to `Units.Kilograms`. This is great to force students to use a property initializer, but we unfortunately don't explain this to the student. We should add a task that explains this behavior and add a test to verify it. I think this should be the second task in the exercise.

### PR DESCRIPTION
**Pull request by 18-F-cali**
Fixes #1644

### CODE REVIEW

File: exercises/concept/weighing-machine/.docs/instructions.md [hunk 1]
```diff
@@ -42,7 +42,18 @@ wm.InputWeight = 60m;
 //  => wm.InputWeight == 60m
 ```
 
-## 2. Ensure that a negative input weight is rejected
+## 2. Ensure that a Default unit is kilograms
```
Comment (ErikSchierboom) [inline exercises/concept/weighing-machine/.docs/instructions.md:45]:
```suggestion
## 2. Allow specifying the unit used by the weighing machine
```

File: exercises/concept/weighing-machine/.docs/instructions.md [hunk 2]
```diff
@@ -42,7 +42,18 @@ wm.InputWeight = 60m;
...
+
+Implement the `WeigingMachine.Unit` property to allow the unit to be Kilograms by default:
```
Comment (ErikSchierboom) [inline exercises/concept/weighing-machine/.docs/instructions.md:47]:
```suggestion
Implement the `WeigingMachine.Unit` property to allow the unit to be set. `Unit.Kilograms` should be the default value:
```

File: exercises/concept/weighing-machine/WeighingMachineTests.cs [hunk 3]
```diff
@@ -79,4 +79,12 @@ public void Apply_large_tare_adjustment_to_allow_negative_display_weight()
         wm.TareAdjustment = 110;
         Assert.Equal(-10, wm.DisplayWeight);
     }
+
+    [Fact(Skip = "Remove this Skip property to run this test")]
+    public void Default_Unit_Is_Kilogram()
```
Comment (ErikSchierboom) [inline exercises/concept/weighing-machine/WeighingMachineTests.cs:84]: Could you move this test up after the first `InputWeight` tests?

**Review by ErikSchierboom — COMMENTED** [review]
Great! Just a couple of nits

**Comment by 18-F-cali** [comment]
So sorry.I accidently merged with main.

File: exercises/concept/weighing-machine/WeighingMachineTests.cs [hunk 4]
```diff
@@ -79,4 +87,6 @@ public void Apply_large_tare_adjustment_to_allow_negative_display_weight()
         wm.TareAdjustment = 110;
         Assert.Equal(-10, wm.DisplayWeight);
     }
+
+
```
Comment (ErikSchierboom) [inline exercises/concept/weighing-machine/WeighingMachineTests.cs:91]:
```suggestion   
```

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Merged. Thanks!

**Pull request merged**